build.sbt

```diff
@@ -826,9 +826,11 @@ lazy val `java-pingpong` = (project in file("samples/java-pingpong"))
 
 lazy val `scala-shopping-cart` = (project in file("samples/scala-shopping-cart"))
   .dependsOn(`scala-support`)
-  .enablePlugins(AkkaGrpcPlugin)
+  .enablePlugins(AkkaGrpcPlugin, DockerPlugin, JavaAppPackaging)
   .settings(
     name := "scala-shopping-cart",
+    dockerSettings,
```

Comment:
Oh, now I see ... those `dockerSettings` come form the `DockerPlugin`, right? And the system properties can be used to override, right?

Reply:
Yea, that is how I understand it to work.

Reply:
So you are invoking `sbt -Ddocker.registry=...`?

Reply:
yes, just like the example provided

```
sbt scala-shopping-cart/docker:publish -Ddocker.registry=<MY-REGISTRY> -Ddocker.username=<MY-REPOSITORY>```

Reply:
Got it. Thanks!
I did not notice that the `dockerSettings` are defined to take system properties (lines 212 and 213).